### PR TITLE
Test resources

### PR DESCRIPTION
**Pull request by jdesrosiers**
Moved resources from "public" to "src/test/resources".

### CODE REVIEW

**Comment by jdesrosiers** [comment]
@zacholauson please review when you get a chance.

File: src/main/java/org/cobspec/CobSpec.java [hunk 1]
```diff
@@ -53,7 +53,7 @@ public static void main(String[] args) throws IOException {
 
         app.get("/method_options", (request) -> Response.create());
         app.post("/method_options", (request) -> Response.create());
-        app.put("/method_options", fileSystemController::write);
+        app.put("/method_options", (request) -> Response.create());
```
Comment (zacholauson) [inline src/main/java/org/cobspec/CobSpec.java:56]: What was the reasoning for this change?

Comment (jdesrosiers) [reply src/main/java/org/cobspec/CobSpec.java:56]: It's just to conform to cob_spec.  It makes this request, but if a resource is created (like it should for a PUT request) another test fails.  The method_options routes aren't intended to do anything.  They only exist to show that an OPTIONS request correctly indicates methods that are supported (even if those methods don't behave correctly).

There are several instances of cob_spec not following the HTTP spec perfectly.  Both of the PRs I submitted on cob_spec address similar issues.

Comment (zacholauson) [reply src/main/java/org/cobspec/CobSpec.java:56]: 👍

**Pull request merged**